nvcc failes to rebuild Docker image, breaking CI

https://github.com/DARMA-tasking/magistrate/actions/runs/5459636748/jobs/9935835735?pr=290

```
Run $(./scripts/check_containers.sh) ubuntu-cpp
  $(./scripts/check_containers.sh) ubuntu-cpp
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    REPO: ***/checkpoint
    ARCH: amd64
    UBUNTU: 20.04
    COMPILER_TYPE: nvidia
    COMPILER: 11.2.0
    BUILD_TYPE: release
    ULIMIT_CORE: 0
    CODE_COVERAGE: 0
    CHECKPOINT_TESTS: 1
    CHECKPOINT_EXAMPLES: 1
    CHECKPOINT_MPI: 1
    CHECKPOINT_WARNINGS_AS_ERRORS: 1
    CHECKPOINT_DOCS: 0
    CHECKPOINT_ASAN: 0
    CHECKPOINT_UBSAN: 0
    CHECKPOINT_SERIALIZATION_ERROR_CHECKING: 0
    CACHE: ~/.local/cache/
    BUILD_ROOT: ~/.local/cache/amd64-ubuntu-20.04-11.2.0-cache
    timestamp: 2023-07-05-01;39;13
Files modified in the latest pull request:
ci/docker/ubuntu-18.04-clang-cpp.dockerfile
ci/docker/ubuntu-18.04-gnu-cpp.dockerfile
ci/docker/ubuntu-18.04-intel-cpp.dockerfile
ci/docker/ubuntu-20.04-gnu-docs.dockerfile
ci/docker/ubuntu-20.04-nvidia-cpp.dockerfile
ci/docker/ubuntu-22.04-clang-cpp.dockerfile
src/CMakeLists.txt
src/checkpoint/container/view_serialize.h
Building ubuntu-cpp
Sending build context to Docker daemon  1.076MB

Step 1/26 : ARG compiler=11.2.0
Step 2/26 : ARG arch=amd64
Step 3/26 : ARG ubuntu=20.04
Step 4/26 : FROM --platform=${arch} nvidia/cuda:${compiler}-devel-ubuntu${ubuntu} as base
manifest for nvidia/cuda:11.2.0-devel-ubuntu20.04 not found: manifest unknown: manifest unknown
Service 'ubuntu-cpp' failed to build : Build failed
Error: Process completed with exit code 1.
```

_Originally posted by @PhilMiller in https://github.com/DARMA-tasking/magistrate/issues/290#issuecomment-1620930177_

--- a/.github/workflows/dockerimage-nvcc-11-ubuntu-mpich.yml
+++ b/.github/workflows/dockerimage-nvcc-11-ubuntu-mpich.yml
@@ -22,7 +22,7 @@ jobs:
       ARCH: amd64
       UBUNTU: 20.04
       COMPILER_TYPE: nvidia
-      COMPILER: 11.2.0
+      COMPILER: 11.2.2
       BUILD_TYPE: release
       ULIMIT_CORE: 0
       CODE_COVERAGE: 0

--- a/ci/docker/ubuntu-20.04-nvidia-cpp.dockerfile
+++ b/ci/docker/ubuntu-20.04-nvidia-cpp.dockerfile
@@ -1,5 +1,5 @@
 
-ARG compiler=11.2.0
+ARG compiler=11.2.2
 ARG arch=amd64
 ARG ubuntu=20.04
 FROM --platform=${arch} nvidia/cuda:${compiler}-devel-ubuntu${ubuntu} as base

--- a/scripts/workflows.ini
+++ b/scripts/workflows.ini
@@ -101,7 +101,7 @@ distro = 22.04
 [PR-tests-nvidia-11]
 test_configuration = "nvcc-11.2, ubuntu, mpich"
 compiler_type = nvidia
-compiler = 11.2.0
+compiler = 11.2.2
 cache_name = "[% linux %]-nvcc-[% compiler %]-cache"
 output_name = .github/workflows/dockerimage-nvcc-11-ubuntu-mpich.yml
 checkpoint_asan = 0

